templates/terraform/constants/bigquery_dataset_access.go

```diff
@@ -0,0 +1,12 @@
+var bigqueryAccessRoleToPrimitiveMap =  map[string]string {
+    "roles/bigQuery.dataOwner": "OWNER",
+    "roles/bigQuery.dataEditor": "EDITOR",
```

Comment:
Shouldn't it be `WRITER` as said in [official doc](https://cloud.google.com/bigquery/docs/access-control-primitive-roles#dataset-primitive-roles)?
```suggestion
    "roles/bigQuery.dataEditor": "WRITER",
```

Reply:
👍